### ISSUE TITLE
Investigate indexing failures

### ISSUE THREAD

**Issue by christinach** [pulibrary/marc_liberation#872]
There are records from October ~~August and September~~ that didn't get indexed. The workaround was to index them manually as users were reporting them. Investigate this.
The records I tracked down after a user reported them in a snap ticket were last update in Voyager:
2020-10-15 23:50      bib: 11962439
2020-10-16 19:52:03 bib: 12052285
2020-10-16 19:52:03 bib: 12052289
2020-10-20 19:59      bib: 12105568

**Comment by christinach** [comment]
Based on the current schedules we are getting the latest changes from voyager at : "10:10am", "2:10pm", "10:35pm" 
https://github.com/pulibrary/marc_liberation/blob/356503c58ab51eb7dcf53f79b190dd7cb01f05de/config/schedule.rb#L32.

We index the latest changes at :"12:40am", "10:55am", "2:55pm" 
 https://github.com/pulibrary/marc_liberation/blob/356503c58ab51eb7dcf53f79b190dd7cb01f05de/config/schedule.rb#L56.
Look into solr failures between 12:00am and 1am.

**Comment by christinach** [comment]
This is related to https://github.com/pulibrary/marc_liberation/issues/874. Since we migrated from mysql to postgres the events page had a random order but not in a specific pattern. We relied on the event_controller index action to list all the events by Event.all. In postgres a select will return all the records but in a random order. We fixed this by forcing an ascending order on the events based on the start attribute of the event. The liberate rake tasks when they pull the json from the events page are checking the start attribute.
